cmd/config.go

```diff
@@ -164,7 +164,6 @@ type Config struct {
 	PA PAConfig
 
 	Common struct {
-		BaseURL string
```

Comment:
Actually, it's a step worse: Current Boulder doesn't even use the WFE.BaseURL field, only the Common.BaseURL field.